frontend/src/views/applications/ServiceDeliveryView.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <v-form ref="form">
+    <h2 class="mb-4">Service Delivery Details</h2>
+    <h4 class="mb-2">
+      Your facility:
+      <span class="facility-name ml-6">{{ currentApplication?.facilityName }}</span>
+    </h4>
+    <v-row v-if="!loading && licences?.length > 0" no-gutters class="mb-2">
+      <v-col cols="12" align="right">
+        <AppButton v-if="isEmpty(panel)" id="expand-button" :primary="false" size="large" width="200px" @click="togglePanel">
+          <v-icon>mdi-arrow-expand-vertical</v-icon>
+          Expand All
+        </AppButton>
+        <AppButton v-else id="collapse-button" :primary="false" size="large" width="200px" @click="togglePanel">
+          <v-icon>mdi-arrow-collapse-vertical</v-icon>
+          Collapse All
+        </AppButton>
+      </v-col>
+    </v-row>
+    <div>
+      <v-skeleton-loader :loading="loading" type="table-tbody">
+        <v-expansion-panels v-model="panel" multiple>
+          <v-expansion-panel v-for="licence in licences" :key="licence.licenceId" :value="licence.licenceId">
+            <v-expansion-panel-title class="header-label">
+              <LicenceHeader :licence="licence" />
+            </v-expansion-panel-title>
+            <v-expansion-panel-text>
+              <h4 class="mb-2 text-decoration-underline">Category Details</h4>
+              <LicenceDetails :licenceDetails="licence.licenceDetails" />
+            </v-expansion-panel-text>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </v-skeleton-loader>
+    </div>
+    <v-checkbox
+      v-model="licenceDeclaration"
+      :value="1"
+      :false-value="0"
+      :true-value="1"
+      color="primary"
+      :rules="rules.required"
+      :disabled="readonly"
+      label="I confirm that the above information is correct."
+      class="mt-4"></v-checkbox>
+    <p>
+      Your organization account manager can update licence details in
+      <router-link :to="{ name: 'manage-facility', params: { facilityId: currentApplication?.facilityId } }">Account Management</router-link>
```

Comment:
Do we want a trailing period on here? For readability it should be consistent with the "I confirm" statement. So either both with or both without.

Reply:
as discussed, since the <router-link> auto add a space between the "Account Management" and ".", the easiest way is to remove the "."